cmd/secret-vault/plugin_test.go

```diff
@@ -9,15 +9,15 @@ import (
 	"github.com/go-vela/secret-vault/vault"
 )
 
-func TestVault_Plugin_Exec(t *testing.T) {
-	// TODO write this test
-}
+//func TestVault_Plugin_Exec(t *testing.T) {
+// TODO write this test
+//}
```

Comment:
minor: what about just yanking these lines? test coverage already tells us what is not covered. alternatively, we write the tests :disappear:

Reply:
ope I did not write that, it was already in the code the I pulled it down so I will delete those